style.css

```diff
@@ -322,6 +322,60 @@ body {
   font-size: 15px;
 }
 
+/* Footer */
+.contact-form {
+  color: #fff;
+  background-color: #6070ff;
+  background-image: url(./assets/header-shapes-mobile.png);
+  background-position: right;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 90px 24px;
+}
+
```

Comment:
@AbeeraTahir Hello! Thanks for all your comments 😃 I corrected all errors except for the bakground image which doesn't look exactly as the figma design and I don't quite now how to fix it (even the colors are different, but I downloaded what Figma has). I'm having trouble with the  font size from the h2 "Contact me". I guess some code is overriding it, but can't figure out which part is the one with the problem. Any suggestions?

Reply:
Hi @montsecruz9, I had a look at your code just now with your new commits and I just figured out one thing! In your index.html, you are targeting the contact form with the `contact-form` class, as shown below:

```
<section class="contact-form">
<h2 class="title-form">Contact me</h2>
```

And you are targeting this `h2` element in your CSS file like this:

```
.title-form h2 {
  font-family: "Poppins", sans-serif;
  font-size: 40px;
  font-weight: 700;
  line-height: 52px;
  letter-spacing: 0;
  text-align: center;
  height: 56px;
  width: 801px;
  left: -213px;
  top: 4734px;
}
```
Remember, if you have given your element a separate class then you have to target it by that class, so your code should be like this:

```
.title-form {
  font-family: "Poppins", sans-serif;
  font-size: 40px;
  font-weight: 700;
  line-height: 52px;
  letter-spacing: 0;
  text-align: center;
  height: 56px;
  width: 801px;
  left: -213px;
  top: 4734px;
}
```

As `h2` is that specific element, which has given the class of `title-form`, and not its child. I hope it will help. 😊